src/main/java/com/faforever/client/game/GameTileController.java

```diff
@@ -110,6 +112,8 @@ public void setGame(Game game) {
     ));
 
     lockIconLabel.visibleProperty().bind(game.passwordProtectedProperty());
+    friendsLabel.visibleProperty().bind(new SimpleBooleanProperty(!playerService.friendsInGame(game).isEmpty()));
```

Comment:
Is that updated if a friend joins a game?

Reply:
yes, you can follow friends when they hop from lobby to lobby

Reply:
my concern was if setGame is called again if a new player joins or if you add a new friend... the later I doubt

Reply:
my concern was if setGame is called again if a new player joins or if you add a new friend... the later I doubt

Reply:
And binding makes no sense if the BooleanProperty
 is never updated!!!